views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/homepage.css

```diff
@@ -0,0 +1,171 @@
+/* Styling for default Primo components */
+
+md-content.md-primoExplore-theme.layout-align-center-start {
+  background-color: var(--color-white);
+  justify-content: center;
+}
+main {
+  max-width: 1040px;
+  margin: 64px auto;
+}
+prm-static.md-padding.flex {
+  padding: 0;
+}
+/* Hide side "spacer" divs in desktop view */
+@media (min-width: 1280px) {
+  prm-search md-content .flex-lg-25 {
+    display: none;
+  }
+}
+@media (min-width: 1280px) {
+  prm-search md-content .flex-lg-10 {
+    display: none;
```

Comment:
@ztucker4 The margin problems have been fixed, except... on the "Journal Title" page, the left margin is present, but empty of the "Journals by category" list.  That list is present in the [deployed `PRIMO_REDESIGN` view`(https://search.library.ucla.edu/discovery/jsearch?vid=01UCS_LAL:PRIMO_REDESIGN) and I see it when I check out the `main` branch locally, but not with this branch.  The culprit seems to be this `display: none;`.

Here's what I see with this branch:
![primo_redesign_missing_journal_categories](https://github.com/UCLALibrary/primo_ve/assets/2213836/d70b164c-daa4-4926-b31c-213c61a62dc9)